pkg/cluster/internal/kubeadm/config.go

```diff
@@ -99,6 +101,11 @@ controlPlaneEndpoint: "{{ .ControlPlaneEndpoint }}"
 api:
   advertiseAddress: "{{ .NodeAddress }}"
   bindPort: {{.APIBindPort}}
+{{ if .PodSecurityPolicy -}}
+# enable PodSecurityPolicy admission controller
+apiServerExtraArgs:
+  enable-admission-plugins: PodSecurityPolicy
```

Comment:
for the newer versions, this will stomp the NodeRestriction controller entry, but it's not really required for the kind use case.

Reply:
hmm that's not great though ....
really we should consider reworking the config to use patches internally. (and possibly making patches less annoying ...)

Reply:
I think that´s a great idea, I'm always afraid of breaking things when modifying this file and adding new fields